web/client/plugins/MapSearch.jsx

```diff
@@ -22,21 +22,11 @@ const ConfigUtils = require('../utils/ConfigUtils');
 <br>if true and with text => search is substituted with remove
 <br>if false and without text => you see only search
 <br>if false and with text => you see both (search and remove)
```

Comment:
please add this one since it is a configurable flag used in the map search
```suggestion
<br>if false and with text => you see both (search and remove)
* @param {boolean} [showContextSearchOption=true] used to show context search option
```